lib/plugins/aws/invokeLocal/index.js

```diff
@@ -52,10 +52,31 @@ class AwsInvokeLocal {
   }
 
   loadEnvVars() {
+    const lambdaName = this.options.functionObj.name;
+    const MemorySize = Number(this.options.functionObj.memorySize)
+      || Number(this.serverless.service.provider.memorySize)
+      || 1024;
+
+    const lambdaDefaultEnvVars = {
+      PATH: '/usr/local/lib64/node-v4.3.x/bin:/usr/local/bin:/usr/bin/:/bin',
+      LANG: 'en_US.UTF-8',
+      LD_LIBRARY_PATH: '/usr/local/lib64/node-v4.3.x/lib:/lib64:/usr/lib64:/var/runtime:/var/runtime/lib:/var/task:/var/task/lib', // eslint-disable-line max-len
+      LAMBDA_TASK_ROOT: '/var/task',
+      LAMBDA_RUNTIME_DIR: '/var/runtime',
+      AWS_REGION: this.options.region || 'us-east-1',
```

Comment:
yeah yeah, what I mean is that `this.options.region` should be set to default already. You shouldn't set default value here because that means that all plugins need to do the same, which would be weird.